README-mux.md

```diff
@@ -0,0 +1,327 @@
+openshift-ansible-callback-plugins openshift-ansible-filter-plugins
+openshift-ansible-lookup-plugins openshift-ansible-playbooks
+openshift-ansible-roles
+
+    # yum install openshift-ansible \
+      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+      openshift-ansible-roles
+
+If the 3.5/1.5 versions of these packages are not available, you can use the
+git repo `https://github.com/openshift/openshift-ansible.git` and the
+`release-1.5` branch:
+
+    # git clone https://github.com/openshift/openshift-ansible.git -b release-1.5
+
+You will need to use the `ansible-playbook` command with an Ansible inventory
+file and a `vars.yaml` file.  You should not have to edit the inventory file.
+All customization can be done via the `vars.yaml` file.
+
+Download the files [vars.yaml.template](vars.yaml.template) and
+[ansible-inventory-origin-15-aio](ansible-inventory-origin-15-aio)
+
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-15-aio > ansible-inventory-origin-15-aio
+
+Copy `vars.yaml.template` to `vars.yaml`.  You will need to change the
+following fields in `vars.yaml`:
+
+* `ansible_ssh_user` - this is the user created in
+  [provisioning](#provisioning-a-machine-to-run-viaq) which can use
+  passwordless ssh
+* `ansible_become` - use `no` if `ansible_ssh_user` is `root`, otherwise,
+  use `yes`
+* `openshift_master_default_subdomain` - this is the public subdomain to use
+  for all of the external facing logging services, such as the OpenShift UI,
+  Kibana, mux, and Elasticsearch.  By default, the OpenShift UI will be
+  accessed via `https://openshift.{{ openshift_master_default_subdomain }}:8443`,
+  Kibana will be at `https://kibana.{{ openshift_master_default_subdomain }}`,
+  etc.
+* `openshift_public_ip` - this is the public IP address, the IP address used in
+  your internal DNS or host look up for browsers and other external client
+  programs.  For example, in OpenStack, this will be the *floating ip* address
+  of the machine.  This may be the same as the `eth0` IP addrss of the machine,
+  in which case, just use `"{{ ansible_eth0.ipv4.address }}"` as the value as
+  is done for `openshift_ip`
+* `openshift_hostname` - this is the private hostname of the machine that will
+  be used inside the cluster.  For example, OpenStack machines will have a
+  "private" hostname assigned by Neutron networking.  This may be the same as
+  the external hostname if you do not have a "private" hostname - in that case,
+  just use `openshift.{{ openshift_master_default_subdomain }}`
+
+You can also override variables in the inventory by setting them in `vars.yaml`.
+
+The public hostname would typically be a DNS entry for the
+public IP address, but you can "fake" these out with `/etc/hosts` entries:
```

Comment:
I suggest adding that `/etc/hosts` needs to be changed on the client machines that will send logs to ES or from which Kibana will be accessed